packages/react/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -56,7 +56,6 @@ interface ThirdwebProviderProps<TChains extends Chain[]>
 export const ThirdwebProvider = <
   TChains extends Chain[] = typeof defaultChains,
 >({
-  clientId: clientId,
```

Comment:
No need to do this here, clientId will be passed down as part of `{...restProps}`